### PR TITLE
MAUI-3102 UG content for .NET MAUI Autocomplete and ComboBox control features

### PR DESCRIPTION
**Pull request by anandhan-rajagopal**
Prepared the UG content for .NET MAUI Autocomplete and ComboBox control  following features.

##Autocomplete:

 - Migration
 - Selection
 - Searching and Filtering
 - UI Customization

##ComboBox:

- Migration
- Editing
- Selection
- Searching
- UI Customization

### CODE REVIEW

File: MAUI/Autocomplete/Migration.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfAutocomplete | Syncfusion 
+description: Learn here all about Migrating from Syncfusion Xamarin Autocomplete to Syncfusion .NET MAUI Autocomplete control and more.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Migration.md:4]: description: Learn all about Migrating from Syncfusion Xamarin Autocomplete to Syncfusion .NET MAUI Autocomplete control and more here.

File: MAUI/Autocomplete/Migration.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+platform: maui
+control: SfAutocomplete
+documentation: ug
+---  
+
+# Migrating from Xamarin SfAutoComplete to .NET MAUI SfAutocomplete 
+
+To migrate easier from Xamarin SfAutoComplete to .NET MAUI `SfAutocomplete,` we kept most of the APIs from Xamarin SfAutoComplete in MAUI SfAutocomplete. However, to maintain the consistency of API naming in MAUI SfAutocomplete, we renamed some of the APIs. The APIs that have been changed in MAUI SfAutocomplete from Xamarin SfAutoComplete are detailed as follows.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Migration.md:12]: To migrate easier from Xamarin SfAutoComplete to .NET MAUI `SfAutocomplete,` we kept most of the APIs from Xamarin SfAutoComplete in MAUI SfAutocomplete. However, to maintain the consistency of API naming in MAUI SfAutocomplete, we renamed some of the APIs. The APIs changed in MAUI SfAutocomplete from Xamarin SfAutoComplete are detailed as follows:

File: MAUI/Autocomplete/Searching-Filtering.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+{% endhighlight %}
+
+{% highlight C# %}
+
+autocomplete.DisplayMemberPath = "Name";
+
+{% endhighlight %}
+{% endtabs %}
+
+For e.g. After typing `T` in selection box, social medias which have starting letter `T` will be listed in drop-down.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Searching-Filtering.md:84]: For e.g. After typing `T` in the selection box, social media which have to start with the letter `T` will be listed in the drop-down.

File: MAUI/Autocomplete/Searching-Filtering.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+The `DisplayMemberPath` and `TextMemberPath` properties of Autocomplete control specify the property path, by which the searching must be done when a custom data is bound to the `ItemsSource` property.
+
+ * `DisplayMemberPath` - Specifies the property path whose value is displayed as text in the drop-down menu. The default value is `string.Empty`.
+
+ * `TextMemberPath` - Specifies the property path whose value is used to perform searching based on user input received in the selection box portion of the `Autocomplete` control. The default value is `string.Empty`. When `TextMemberPath` is `null` or `string.Empty`, searching will be performed based on `DisplayMemberPath`. 
+
+N> `DisplayMemberPath` and `TextMemberPath` will be effective for the collection item that holds two or more properties in it.
+
+N> When both the `DisplayMemberPath` and `TextMemberPath` properties have a `null` or `string.Empty` value, searching will be performed based on the class name with namespace of the item.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Searching-Filtering.md:24]: N> When both the `DisplayMemberPath` and `TextMemberPath` properties have a `null` or `string.Empty` value, searching will be performed based on the class name with the namespace of the item.

File: MAUI/Autocomplete/Searching-Filtering.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+
+For e.g. After typing `T` in selection box, social medias which have starting letter `T` will be listed in drop-down.
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete search based on display member path](Images/Searching_Filtering/SearchBasedDisplayMemberPath.png)
+
+### Searching based on TextMemberPath
+
+Searching will be performed based on the `TextMemberPath` property, while entering the text into the selection box. If `TextMemberPath` is `null` or `string.Empty`, searching will be performed based on `DisplayMemberPath`.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Searching-Filtering.md:92]: Searching will be performed based on the `TextMemberPath` property while entering the text into the selection box. If `TextMemberPath` is `null` or `string.Empty`, searching will be performed based on `DisplayMemberPath`.

File: MAUI/Autocomplete/Searching-Filtering.md [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete filtering based on started text](Images/Searching_Filtering/SearchModeStartsWith.png)
+
+### Filter with contains text
+
+Set the `TextSearchMode` property value to `Contains` to filter the matching items based on the containing specific text. The first item  in the drop-down list that matches the user input will be highlighted.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Searching-Filtering.md:153]: Set the `TextSearchMode` property value to `Contains` to filter the matching items based on the containing specific text. The first item in the drop-down list that matches the user input will be highlighted.

File: MAUI/Autocomplete/Selection.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: Selection in .NET MAUI Autocomplete control | Syncfusion
+description: Learn here all about Selection support in Syncfusion .NET MAUI Autocomplete (SfAutocomplete) control and more.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Selection.md:4]: description: Learn all about Selection support in Syncfusion .NET MAUI Autocomplete (SfAutocomplete) control and more here.

File: MAUI/Autocomplete/Selection.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+documentation: ug
+---
+
+# Selection in .NET MAUI Autocomplete (SfAutocomplete)
+
+The `Autocomplete` allows user to select a single item from dropdown list.
+
+## Single selection
+
+The `Autocomplete` allows user to select an item by entering the value using keyboard, then selecting from the drop-down list by clicking the item. The selected item can be retrieved from the `SelectedItem` property.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Selection.md:16]: The `Autocomplete` allows the user to select an item by entering the value using the keyboard, then selecting from the drop-down list by clicking the item. The selected item can be retrieved from the `SelectedItem` property.

File: MAUI/Autocomplete/Selection.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete with single selection mode](Images/Selection/SingleSelection.png)
+
+## Selection changed notification
+
+When an item is selected from the drop-down list, the `SelectionChanged` event is triggered. The `SelectionChanged` event contains the newly selected and previously selected item in the `CurrentSelection` and `PreviousSelection` properties. The `SelectionChanged` contains the following properties:
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Selection.md:84]: When an item is selected from the drop-down list, the `SelectionChanged` event is triggered. The `SelectionChanged` event contains the newly selected and previously selected items in the `CurrentSelection` and `PreviousSelection` properties. The `SelectionChanged` contains the following properties:

File: MAUI/Autocomplete/Selection.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+{% endhighlight %}
+
+{% highlight C# %}
+
+autocomplete.SelectionChanged += OnSelectionChanged;
+
+{% endhighlight %}
+{% endtabs %}
+
+The `SelectionChanged` event can be handled as follows.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/Selection.md:109]: The `SelectionChanged` event can be handled as follows:

File: MAUI/Autocomplete/UI-Customization.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: UI Customization in .NET MAUI Autocomplete control | Syncfusion
+description: Learn here all about UI customization support in Syncfusion .NET MAUI Autocomplete control into .NET MAUI application and its features.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/UI-Customization.md:4]: description: Learn all about UI customization support in Syncfusion .NET MAUI Autocomplete control into .NET MAUI application and its features here.

File: MAUI/Autocomplete/UI-Customization.md [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+platform: maui
+control: SfAutocomplete
+documentation: ug
+---
+
+# UI Customization in .NET MAUI Autocomplete (SfAutocomplete)
+
+This section explains different UI customization available in the `Autocomplete` control.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/UI-Customization.md:12]: This section explains different UI customizations available in the `Autocomplete control.

File: MAUI/Autocomplete/UI-Customization.md [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+
+## Placeholder
+
+You can prompt the user with any information by using the `Placeholder` property. This text will be displayed only if no items are selected or the edit text is empty. The default value of `Placeholder` property is `string.Empty` (No string will be displayed).
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/UI-Customization.md:16]: You can prompt the user with any information by using the `Placeholder` property. This text will be displayed only if no items are selected or the edit text is empty. The default value of the `Placeholder` property is `string.Empty` (No string will be displayed).

File: MAUI/Autocomplete/UI-Customization.md [hunk 14]
```diff
@@ -0,0 +1,154 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete placeholder](Images/UICustomization/Placeholder.png)
+
+## Placeholder Color
+
+The placeholder text color can be changed by using the `PlaceholderColor` property. The default value of `PlaceholderColor` property is `Colors.Gray`.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/UI-Customization.md:43]: The placeholder text color can be changed by using the `PlaceholderColor` property. The default value of the `PlaceholderColor` property is `Colors.Gray`.

File: MAUI/Autocomplete/UI-Customization.md [hunk 15]
```diff
@@ -0,0 +1,154 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete border color](Images/UICustomization/BorderColor.png)
+
+## Maximum DropDown Height
+
+The maximum height of the drop-down can be changed by using `MaxDropDownHeight` property of `Autocomplete` control. The default value of `MaxDropDownHeight` property is `400d`.
```
Comment (OnyimbiNelson) [inline MAUI/Autocomplete/UI-Customization.md:129]: The maximum height of the drop-down can be changed by using the `MaxDropDownHeight` property of the `Autocomplete` control. The default value of `MaxDropDownHeight` property is `400d`.

File: MAUI/ComboBox/Editing.md [hunk 16]
```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Editing in .NET MAUI ComboBox control | Syncfusion
+description: Learn here all about Editable modes support in Syncfusion .NET MAUI ComboBox (SfComboBox) control and more.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Editing.md:4]: description: Learn all about Editable modes support in Syncfusion .NET MAUI ComboBox (SfComboBox) control and more `here.

File: MAUI/ComboBox/Editing.md [hunk 17]
```diff
@@ -0,0 +1,99 @@
...
+documentation: ug
+---
+
+# Editing in .NET MAUI ComboBox (SfComboBox)
+
+The `ComboBox` control supports both editable and non-editable text boxes for selecting an item from a data source. To enable editing functionality, set `IsEditable` property as `true`. The default value is `false`.
+
+## Editable ComboBox
+
+In editable mode, the `ComboBox` allows users to edit in the text box and it automatically appends the remaining letters to the entered text when it is valid.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Editing.md:16]: In the editable mode, the `ComboBox` allows users to edit in the text box and it automatically appends the remaining letters to the entered text when it is valid.

File: MAUI/ComboBox/Editing.md [hunk 18]
```diff
@@ -0,0 +1,99 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox choose item using editing](Images/Editing/EditableMode.png)
+
+## Non-editable ComboBox
+
+Non-editable mode prevents users from editing and instead allows them to select from drop-down list.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Editing.md:44]: Non-editable mode prevents users from editing and instead allows them to select from the drop-down list.

File: MAUI/ComboBox/Editing.md [hunk 19]
```diff
@@ -0,0 +1,99 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox choose item using non editing mode](Images/Editing/NonEditableMode.png)
+
+## Hide clear button in the editor
+
+By default, the clear button `X` will be displayed in the editor of the `ComboBox` control, which can be used to clear the entered input. Hide the clear button in `ComboBox` control using the `IsClearButtonVisible` property. The default value of `IsClearButtonVisible` property value is `true`.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Editing.md:72]: By default, the clear button `X` will be displayed in the editor of the `ComboBox` control, which can be used to clear the entered input. Hide the clear button in `ComboBox` control using the `IsClearButtonVisible` property. The default value of the `IsClearButtonVisible` property value is `true`.

File: MAUI/ComboBox/Migration.md [hunk 20]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfComboBox | Syncfusion 
+description: Learn here all about Migrating from Syncfusion Xamarin ComboBox to Syncfusion .NET MAUI ComboBox control and more.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Migration.md:4]: description: Learn all about Migrating from Syncfusion Xamarin ComboBox to Syncfusion .NET MAUI ComboBox control and more here.

File: MAUI/ComboBox/Searching.md [hunk 21]
```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: Searching in .NET MAUI ComboBox control | Syncfusion
+description: Learn how to perform searching operation in Syncfusion .NET MAUI ComboBox control and more.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Searching.md:4]: description: Learn how to perform the searching operation in Syncfusion .NET MAUI ComboBox control and more.

File: MAUI/ComboBox/Searching.md [hunk 22]
```diff
@@ -0,0 +1,143 @@
...
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox search based on DisplayMemberPath](Images/Searching/SearchDisplayMemberPath.png)
+
+N> Auto appending of text is supported only in `Editable` mode and `TextSearchMode` property should be `StartsWith`.
+
+## Text Search Mode
+
+The `TextSearchMode` property of the `ComboBox` can be used to regulate how the control behaves when it receives user input. The default text searching type is `StartsWith`, ignoring accent and it is case insensitive. The available text search modes are,
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Searching.md:85]: The `TextSearchMode` property of the `ComboBox` can be used to regulate how the control behaves when it receives user input. The default text searching type is `StartsWith`, ignoring accent and it is case insensitive. The available text search modes are:

File: MAUI/ComboBox/Selection.md [hunk 23]
```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Selection in .NET MAUI ComboBox control | Syncfusion
+description: Learn here all about Selection modes support in Syncfusion .NET MAUI ComboBox (SfComboBox) control and more.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Selection.md:4]: description: Learn all about Selection modes support in Syncfusion .NET MAUI ComboBox (SfComboBox) control and more here.

File: MAUI/ComboBox/Selection.md [hunk 24]
```diff
@@ -0,0 +1,177 @@
...
+platform: maui
+control: SfComboBox
+documentation: ug
+---
+
+# Selection in .NET MAUI ComboBox (SfComboBox)
+
+## Single selection
+
+The `ComboBox` allows user to select an single item from the drop-down list.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Selection.md:14]: The `ComboBox` allows users to select a single item from the drop-down list.

File: MAUI/ComboBox/Selection.md [hunk 25]
```diff
@@ -0,0 +1,177 @@
...
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox UI selection](Images/Selection/UISelection.png)
+
+
+### Programmatic selection 
+
+The selected item can be changed programmatically by using the `SelectedItem` or `SelectedIndex` property of `ComboBox` control.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Selection.md:38]: The selected item can be changed programmatically by using the `SelectedItem` or `SelectedIndex` properties of the `ComboBox` control.

File: MAUI/ComboBox/Selection.md [hunk 26]
```diff
@@ -0,0 +1,177 @@
...
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox programmatic selection](Images/Selection/ProgramaticSelection.png)
+
+
+## Selection changed notification
+
+When an item is selected from the drop-down list, the `SelectionChanged` event is triggered. The `SelectionChanged` event contains the newly selected and previously selected item in the `CurrentSelection` and `PreviousSelection` properties. The `SelectionChanged` contains the following properties:
+
+ * CurrentSelection - Contains the item that were currently selected.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Selection.md:70]: * CurrentSelection - Contains the items that were currently selected.

File: MAUI/ComboBox/Selection.md [hunk 27]
```diff
@@ -0,0 +1,177 @@
...
+ * PreviousSelection - Contains the item that were previously selected.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Selection.md:71]: * PreviousSelection - Contains the items that were previously selected.

File: MAUI/ComboBox/Selection.md [hunk 28]
```diff
@@ -0,0 +1,177 @@
...
+{% endhighlight %}
+
+{% highlight C# %}
+
+comboBox.SelectionChanged += OnSelectionChanged;
+
+{% endhighlight %}
+{% endtabs %}
+
+The SelectionChanged event can be handled as follows.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Selection.md:92]: The SelectionChanged event can be handled as follows:

File: MAUI/ComboBox/Selection.md [hunk 29]
```diff
@@ -0,0 +1,177 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox selected value](Images/Selection/SelectedValuePath.png)
+
+## Open a drop-down programmatically
+
+In `ComboBox` control, the drop-down can be opened or closed programmatically by using the `IsDropDownOpen` property. The default value of `IsDropDownOpen` property is `false`.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/Selection.md:157]: In the `ComboBox` control, the drop-down can be opened or closed programmatically by using the `IsDropDownOpen` property. The default value of the `IsDropDownOpen` property is `false`.

File: MAUI/ComboBox/UI-Customization.md [hunk 30]
```diff
@@ -0,0 +1,176 @@
...
+title: UI Customization in .NET MAUI ComboBox control | Syncfusion
+description: Learn here all about UI customization support in Syncfusion .NET MAUI ComboBox control into .NET MAUI application and its basic features.
+platform: maui
+control: SfComboBox
+documentation: ug
+---
+
+# UI Customization in .NET MAUI ComboBox (SfComboBox)
+
+This section explains different UI customization available in `ComboBox`.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/UI-Customization.md:12]: This section explains different UI customizations available in `ComboBox`.

File: MAUI/ComboBox/UI-Customization.md [hunk 31]
```diff
@@ -0,0 +1,176 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox placeholder color](Images/UICustomization/PlaceholderColor.png)
+
+## Clear Button Icon Color
+
+The clear button icon color can be changed by using the `ClearButtonIconColor` property. The default value of `ClearButtonIconColor` property is `Colors.Black`.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/UI-Customization.md:70]: The clear button icon color can be changed by using the `ClearButtonIconColor` property. The default value of the `ClearButtonIconColor` property is `Colors.Black`.

File: MAUI/ComboBox/UI-Customization.md [hunk 32]
```diff
@@ -0,0 +1,176 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox clear button color](Images/UICustomization/ClearButtonColor.png)
+
+## DropDown Icon Color
+
+The DropDown icon color can be changed by using the `DropDownIconColor` property. The default value of `DropDownIconColor` property is `Colors.Black`.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/UI-Customization.md:96]: The DropDown icon color can be changed by using the `DropDownIconColor` property. The default value of the `DropDownIconColor` property is `Colors.Black`.

File: MAUI/ComboBox/UI-Customization.md [hunk 33]
```diff
@@ -0,0 +1,176 @@
...
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox border color](Images/UICustomization/BorderColor.png)
+
+
+## Maximum DropDown Height
+
+The maximum height of the drop-down can be changed by using the `MaxDropDownHeight` property of `ComboBox` control. The default value of `MaxDropDownHeight` property is `400d`.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/UI-Customization.md:151]: The maximum height of the drop-down can be changed by using the `MaxDropDownHeight` property of the `ComboBox` control. The default value of `MaxDropDownHeight` property is `400d`.

File: MAUI/ComboBox/UI-Customization.md [hunk 34]
```diff
@@ -0,0 +1,176 @@
...
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox border color](Images/UICustomization/BorderColor.png)
+
+
+## Maximum DropDown Height
+
+The maximum height of the drop-down can be changed by using the `MaxDropDownHeight` property of `ComboBox` control. The default value of `MaxDropDownHeight` property is `400d`. 
+
+ N> If the `MaxDropDownHeight` is too small compared to the populated items, scroll viewer will be automatically shown to navigate the hidden items.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/UI-Customization.md:153]: N> If the `MaxDropDownHeight` is too small compared to the populated items, the scroll viewer will be automatically shown to navigate the hidden items.

File: MAUI/ComboBox/UI-Customization.md [hunk 35]
```diff
@@ -0,0 +1,176 @@
...
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox placeholder](Images/UICustomization/Placeholder.png)
+
+
+## Placeholder color
+
+The placeholder texts’ foreground can be changed by using the `PlaceholderColor` property. The default value of `PlaceholderColor` property is `Colors.Gray`.
```
Comment (OnyimbiNelson) [inline MAUI/ComboBox/UI-Customization.md:43]: The placeholder texts’ foreground can be changed by using the `PlaceholderColor` property. The default value of the `PlaceholderColor` property is `Colors.Gray`.

**Review by PaulAndersonS — APPROVED** [review]
Fine

**Pull request merged**